src/modules/Dropdown/Dropdown.js

```diff
@@ -1047,7 +1047,9 @@ export default class Dropdown extends Component {
     } else if (hasValue) {
       _text = _.get(this.getItemByValue(value), 'text')
     }
-
+    if (classes === 'text') {
+      return <div className={classes} role='alert' aria-live='polite'>{_text}</div>
+    }
```

Comment:
@levithomason  we only need to mark as 'alert' a div which text changes and the user should know about it. In the dropdowns case, the dive which displays the value selected by the user.  
To the best of my knowledge, renderText  returns a div which can be marked with classes: text , default and filtered. Looking at the examples of the documentation, I noticed than 'text' is always used to the div which contains the value selected, but I don't know when the other two options are used. Can you give me examples of them to decide if the three cases we need to add the alert tag?